science-platform/templates/sqlproxy-cross-project-application.yaml

```diff
@@ -2,31 +2,28 @@
 apiVersion: v1
 kind: Namespace
 metadata:
-  name: sqlproxy-cross-project
+  name: "sqlproxy-cross-project"
```

Comment:
I don't understand why you're quoting all the strings in this file, when we don't elsewhere, but it seems unlikely to hurt anything.

Reply:
I've been adding quoting to each file in Phalanx when I touch it because it means one doesn't have to remember YAML's quoting rules, but I admit that I'm inconsistent about it. (A bunch of the applications have already been converted to fully quoted, though, so this is making them more consistent.)